_docs/launch.md

```diff
@@ -115,6 +115,39 @@ Consonance's strategy is to provision either on-demand VMs or spot priced VMs de
 consonance run --tool-dockstore-id quay.io/collaboratory/dockstore-tool-bamstats:1.25-6_1.0 --run-descriptor Dockstore.json --flavour <AWS instance-type>
 ```
 
+## Notifications
+The Dockstore CLI has the ability to provide notifications via a Dockstore wrapper HTTP post to a user-defined endpoint for 4 milestones:
```

Comment:
Can be simplified to "The Dockstore CLI has the ability to provide notifications via an HTTP post to a user-defined endpoint for the following steps"

and later 

`Additionally, it will also provide notifications when any of these steps have failed.`